### DISCUSSION TITLE
Badge is always showing "failing"

### DISCUSSION THREAD

**Discussion by badsyntax** [community/community#25846] [Actions]
<p>Hi there. For some reason I can’t figure out, the badge for my workflow is always showing “failing” even through the worflow is not failing. </p>
<p>Here is the badge: <a href="https://github.com/badsyntax/richard-willis-vscode-extension-pack/workflows/Node%20CI/badge.svg" rel="nofollow noopener">https://github.com/badsyntax/richard-willis-vscode-extension-pack/workflows/Node%20CI/badge.svg</a></p>
<p>Here is the workflow: <a href="https://github.com/badsyntax/richard-willis-vscode-extension-pack/actions?workflow=Node+CI" rel="nofollow noopener">https://github.com/badsyntax/richard-willis-vscode-extension-pack/actions?workflow=Node+CI</a></p>
<p>It seems like the badge should be showing the success state. I really can’t figure out why it’s not. </p>
<p>Anyone got any pointers? Thanks in advance.</p>

**Comment by mscoutermarsh** [comment]
<p>Hi <a class="mention" href="/badsyntax">@badsyntax</a>,</p>
<p>I looked into it quickly and don’t see anything wrong with your badge URL. Filed an issue internally to have someone take a look into it.</p>
<p>Thanks for the detailed report! </p>

**Comment by cschleiden** [comment]
<p>Thanks <a class="mention" href="/badsyntax">@badsyntax</a> ,</p>
<p>we’ve made a change to allow filtering by event types and defaulted that filter to the <code>push</code> event. Your recent successful workflow runs were all triggered by the <code>release</code> event, but by default the badge displays one of the older, failing <code>push</code>-triggered runs. </p>
<p>As a quick workaround you could specify the <code>release</code> event explicitly:</p>
<pre><code>https://github.com/badsyntax/richard-willis-vscode-extension-pack/workflows/Node%20CI/badge.svg?event=release
</code></pre>
<p><img src="https://github.com/badsyntax/richard-willis-vscode-extension-pack/workflows/Node%20CI/badge.svg?event=release" alt="https://github.com/badsyntax/richard-willis-vscode-extension-pack/workflows/Node%20CI/badge.svg?event=release"></p>
<p>but I can see how the current behavior is confusing, and we’ll take a look to change the default behavior.</p>
<p>Thanks,</p>
<p> Chris</p>

**Comment by cschleiden** [comment, marked as answer]
<p>We’ve deployed a change to not default to the <code>push</code> event, the badge you posted should now work as expected. </p>
<p>Thanks,</p>
<p> Chris</p>

**Comment by badsyntax** [comment]
<p>Awesome thank you! :slight_smile:</p>
